### DISCUSSION TITLE
GitHub Copilot ✖️  - The age of AI has begun!

### DISCUSSION THREAD

**Discussion by Akash1134** [community/community#51938] [Copilot News and Announcements]
👋 Hello technophiles! Your favourite pair programmer has levelled up 🍄  to give you a preview of the **future of GitHub and AI**. Since we released the world’s first at-scale AI developer tool in 2021, over one million developers have activated [GitHub Copilot](https://github.com/features/copilot). Last month we announced [GitHub Copilot X](https://github.blog/2023-03-22-github-copilot-x-the-ai-powered-developer-experience/) - the next phase of our mark on the future of AI-powered software development.  

If you haven’t taken a [**sneak peek 👀  at the future**](https://github.com/orgs/community/discussions/50713) yet, here’s a recap: 

https://user-images.githubusercontent.com/95834130/228540208-76ad97c6-7d60-43dd-8f00-fb071a9993a3.mp4

Want to read 📚 instead? Check out the [**detailed blog post**](https://github.blog/2023-03-22-github-copilot-x-the-ai-powered-developer-experience/) from GitHub CEO, [Thomas Dohmke](https://github.com/ashtom).

Excited? So are we! 🤩  We’ve had a great time [answering all of your questions](https://github.com/orgs/community/discussions/50727) about **Copilot X**, if you still have any, post them to this [discussion](https://github.com/orgs/community/discussions/50713) or in the comments below. 

## 💌  From the Community - Shoutout 📢  to these members

Let's thank these community members for sharing their knowledge and making the Copilot community a place where anyone can ask their questions whilst learning from other users.

- A huge round of applause for [@Anmol-Baranwal](https://github.com/Anmol-Baranwal) for consistently helping users in the community. Here’s one great example where they provided a detailed solution to [@devluist](https://github.com/devluist) for an issue with [accessing private repos](https://github.com/orgs/community/discussions/49912).  
- We are on a spree with self-solvers in the community. It’s just delightful to see so many members coming back to their posts, sharing a solution and helping so many other users.
   - Awesome job by [@yk-sd](https://github.com/yk-sd) for dedicating time to share your [step-by-step solution](https://github.com/orgs/community/discussions/50484) with the community on how to troubleshoot this issue when running Copilot with VS 2022.
    - Much deserving 👏 for [@AdamBD](https://github.com/AdamBD) for their dedication to getting to the [root cause](https://github.com/orgs/community/discussions/50415) of the issue with NeoVim and Copilot. What an incredible self-solve 💯

---
## Good Reads - Your dose 💊  of AI updates

- [GitHub Galaxy 2023](https://github.blog/2023-03-28-github-galaxy-2023-empower-developer-teams-with-a-new-developer-experience/) -  Learn how GitHub’s one, integrated platform–powered by AI and secure at every step—help developer teams be more productive, collaborative, and efficient. 🛰️ 
- GitHub Copilot for Business is already revolutionizing developer productivity for 5,000 organizations. Just take a look at [Duolingo](https://github.com/customer-stories/duolingo) where Engineering managers reported a **25% increase in developer speed and 67% decrease in median code review turnaround time** with [GitHub Copilot](https://docs.github.com/en/enterprise-cloud@latest/copilot/overview-of-github-copilot/about-github-copilot-for-business) 🚀  . Wanna find out how Copilot for Business can help increase developer productivity in your organization? [@blackgirlbytes](https://github.com/blackgirlbytes) presents this awesome guide 📖 to help you decide [why should your team use Copilot for Business](https://github.com/orgs/community/discussions/49422)?
- [8 things you didn’t know you could do](https://github.blog/2022-09-14-8-things-you-didnt-know-you-could-do-with-github-copilot/) with GitHub Copilot. 🎱 
---
## Your Feedback Matters 🗳️

- Tell us what you think of [github.com/community](https://github.com/orgs/community/discussions): What's working and where can we improve? Share now in our [**Community Survey**](https://github.com/orgs/community/discussions/50532) before April 4th!  📆 
- Need help ❓  with your projects or want to talk about programming tools, concepts, languages, and frameworks? Want some great Career Tips? Explore the all-new [🧑‍💻 Programming Help](https://github.com/orgs/community/discussions/categories/programming-help) category and let us know what you think. 

---
## Resources to 🔖 

* [**An introduction to GitHub Copilot**](https://github.com/orgs/community/discussions/39086) ✔️ 
* [GitHub Copilot Research Recitation](https://github.blog/2021-06-30-github-copilot-research-recitation/) 👨‍🔬 
* [How GitHub Copilot helps improve developer productivity](https://github.blog/2022-07-14-research-how-github-copilot-helps-improve-developer-productivity/) 💯 
* [**Copilot Troubleshooting Resources**](https://docs.github.com/en/copilot/troubleshooting-github-copilot) ⚙️ 
* [Getting Started](https://docs.github.com/en/copilot) with GitHub Copilot  🚥
---
That's it for now! See you in the [**FUTURE**](https://github.com/features/preview/copilot-x)...

**Comment by sy-python** [comment]
The Future - unless you are in a country GitHub does not deem worth existing.

**Reply by Zephyr1369** [reply]
that would b exiting. no "c" lol

**Reply by sy-python** [reply]
@Zephyr1369 what
